packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.yargs.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path'
+
+import fg from 'fast-glob'
+import task, { TaskInnerAPI } from 'tasuku'
+
+import { getPaths } from '@redwoodjs/project-config'
+
+import runTransform from '../../../lib/runTransform'
+
+export const command = 'replace-component-svgs'
+export const description = '(v6.x.x->v6.x.x) Converts world to bazinga'
```

Comment:
Is the version range correct here? I thought it'd be v5 -> v6 or even * -> v6 (or whatever the syntax is for "any version to version 6")
And I don't understand the description text 🙂

Reply:
Thanks!